central/vulnmgmt/vulnerabilityrequest/datastore/datastore_impl.go

```diff
@@ -232,7 +237,8 @@ func (ds *datastoreImpl) UpdateRequestExpiry(ctx context.Context, id, message st
 		req.Status = storage.RequestStatus_APPROVED_PENDING_UPDATE
 	}
 
-	if err := ds.store.Upsert(ctx, req); err != nil {
+	storeWriteCtx := getStoreAccessContext(ctx, storage.Access_READ_WRITE_ACCESS)
```

Comment:
nit:  In some functions you name this `storeReadWriteCtx` and in some `storeWriteCtx`.  I understand in some places the intent may just be write, but since it is actually RW, I think I would always call it `storeReadWriteCtx` for consistency

Reply:
That makes sense. I'll push that in a follow-up PR (possibly in the one handling the migration stores).